src/lib/crud.service.ts

```diff
@@ -26,9 +26,7 @@ export class CrudService<T extends EntityType> {
     constructor(public readonly repository: Repository<T>) {
         this.usableQueryRunner = SUPPORTED_REPLICATION_TYPES.has(this.repository.metadata.connection?.options.type);
         this.primaryKey = this.repository.metadata.primaryColumns?.map((columnMetadata) => columnMetadata.propertyName) ?? [];
-        this.columnNames = this.repository.metadata.columns.map((column) =>
-            column.embeddedMetadata ? column.propertyPath : column.databaseName,
-        );
+        this.columnNames = this.repository.metadata.columns.map((column) => column.propertyPath);
```

Comment:
This line is cause.

`databaseName` is the column name managed in the database.

There is no problem if do not use `namingStrategy` separately.

If `namingStrategy` is applied, `databaseName` will be invalid in the Repository.